docs/changelog.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## 0.1.3 (In Progress)
+  * NEW: `dusty restart --repos <repos>` will restart active containers which use the specified repos.
+  * NEW: `dusty disk backup <destination>` will save the contents of your VM's `/persist` directory
+  * NEW: `dusty disk restore <source>` will write to your VM's `/persist` from `<source>`
```

Comment:
I like the pattern that NEW: 'dusty status' had.  It stated what has been added and what this new command does. Makes it a little more clear that this is a new command.  Not sold one way or the other, thoughts?